packages/netlify-cms-backend-git-gateway/src/GitHubAPI.js

```diff
@@ -45,7 +45,7 @@ export default class API extends GithubAPI {
     return this.tokenPromise().then(jwtToken => {
       const baseHeader = {
         Authorization: `Bearer ${jwtToken}`,
-        'Content-Type': 'application/json',
+        'Content-Type': 'application/json; charset=utf-8',
```

Comment:
We send unicode characters in our commit messages (e.g. `Create {{collection}} “{{slug}}”`).
While this wasn't an issue for GitHub, the recording proxy we use for our tests was recently [updated to use ISO 8859 1](https://github.com/mock-server/mockserver/blob/f48922171dfa4babcd552f9e23282aa4e0fd10c5/mockserver-core/src/main/java/org/mockserver/mappers/ContentTypeMapper.java#L34) as default when the charset is not specified which broke our tests.